platform/ios/src/MGLMapView.mm

```diff
@@ -4605,7 +4606,7 @@ - (void)updateHeadingForDeviceOrientation
     return [self visibleFeaturesAtPoint:point inStyleLayersWithIdentifiers:nil];
 }
 
-- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers
+- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(NS_SET_OF(NSString *) *)styleLayerIdentifiers withPredicate:(NSPredicate *)predicate
```

Comment:
This change needs to be reflected in the header. Otherwise, someone will call `-visibleFeaturesAtPoint:inStyleLayersWithIdentifiers:` and crash because dynamic method dispatch will fail.